File: server/src/main/java/splendor/model/game/Color.java
package splendor.model.game;

/**
 * The colors used in the game.
 */
public enum Color {
  /**
   * White token and colour.
   */
  WHITE,
  /**
   * Green token and colour.
   */
  GREEN,
  /**
   * Blue token and colour.
   */
  BLUE,
  /**
   * Red token and colour.
   */
  RED,
  /**
   * Brown token and colour.
   */
  BROWN,
  /**
   * Gold token and colour.
   */
  GOLD,
  /**
   * Not a token. Only used as a colour (for example, player's colour to be used in the game)
   */
  YELLOW;

  /**
   * The colors of tokens.
   *
   * @return the colors of tokens
   */
  public static Color[] tokenColors() {
    return new Color[] {WHITE, GREEN, BLUE, RED, BROWN, GOLD};
  }
}

File: client/game/color.py
from enum import Enum


class Color(Enum):
    BLUE = (0, 0, 255)
    GREEN = (0, 255, 0)
    RED = (255, 0, 0)
    YELLOW = (255, 255, 0)
    BROWN = (139, 69, 19)
    WHITE = (255, 255, 255)
    GOLD = (255, 215, 0)
    BLACK = (0, 0, 0)
    GREY = (128, 128, 128)

    @classmethod
    def get_token_colors(cls):
        return [cls.WHITE,cls.BLUE,cls.GREEN,cls.RED,cls.BROWN,cls.GOLD]